example/ios/Runner/AppDelegate.swift

```diff
@@ -10,62 +10,11 @@ import Branch
         _ application: UIApplication,
         didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?
         ) -> Bool {
-        GeneratedPluginRegistrant.register(with: self)
-        guard let controller = window?.rootViewController as? FlutterViewController else {
-            fatalError("rootViewController cannot be casted to FlutterViewController")
-        }
-        
-        let eventChannel = FlutterEventChannel(name: "flutter_branch_io/event", binaryMessenger: controller)
-
-        eventChannel.setStreamHandler(self)
-        
         // if you are using the TEST key
         // WARNING: DELETE THIS LINE FOR PRODUCTION USE
         Branch.setUseTestBranchKey(true)
-        // listener for Branch Deep Link data
-        Branch.getInstance().initSession(launchOptions: launchOptions) { (params, error) in
-            // do stuff with deep link data (nav to page, display content, etc)
-            print(params as? [String: AnyObject] ?? {})
-            if (self.eventSink != nil) {
-                do {
-                    let jsonData = try JSONSerialization.data(withJSONObject: params, options: .prettyPrinted)
-                    let jsonString = String(data: jsonData, encoding: String.Encoding.utf8)
-                    self.eventSink!(jsonString)
-                } catch {
-                    print("BRANCH IO FLUTTER IOS ERROR")
-                    print(error)
-                }
-            } else {
-                print("Branch IO eventSink is nil")
-            }
-        }
-        
+        GeneratedPluginRegistrant.register(with: self)
         return super.application(application, didFinishLaunchingWithOptions: launchOptions)
     }
     
-    override func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
-        Branch.getInstance().application(app, open: url, options: options)
-        return true
-    }
-    
-    func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
-        // handler for Universal Links
-        Branch.getInstance().continue(userActivity)
-        return true
-    }
-    
-    override func application(_ application: UIApplication, didReceiveRemoteNotification userInfo: [AnyHashable : Any], fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
-        // handler for Push Notifications
-        Branch.getInstance().handlePushNotification(userInfo)
-    }
-    
-    func onListen(withArguments arguments: Any?, eventSink events: @escaping FlutterEventSink) -> FlutterError? {
```

Comment:
Please also delete the `FlutterStreamHandler` identifier on the class declaration

Reply:
and please also delete the `FlutterEventSink` object inside the class

Reply:
fix by https://github.com/blackmenthor/flutter-branch-io/pull/22/commits/35e096749cf87691009e2ce20235c5d9018a6174